src/EnergyPlus/CurveManager.cc

```diff
@@ -458,7 +460,7 @@ namespace CurveManager {
 		int NumTables; // Total tables in the input file
 		int CurveIndex; // do loop index
 		int CurveNum; // current curve number
-		Array1D_string Alphas( 13 ); // Alpha items for object
+		Array1D_string Alphas( 14 ); // Alpha items for object
```

Comment:
Alphas needed to be extended by 1 for the unit type field in the multivariable lookup objects.  I didn't increment Numbers up from 10000 to 10002, but I can.  But I probably won't.